web/frontend/src/language/fr.json

```
@@ -75,49 +75,49 @@
     "addUser": "Ajouter un utilisateur",
     "role": "Rôle",
     "roles": "Rôles",
-    "edit": "Editer",
-    "nothingToAdd": "Il y a rien à ajouter.",
+    "edit": "Éditer",
+    "nothingToAdd": "Il n'y a rien à ajouter.",
     "duplicateCandidate": "Ce candidat a déjà été ajouté.",
     "add": "Ajouter",
     "exportJSON": "Exporter en JSON",
     "delete": "Supprimer",
-    "addVoters": "Ajouter électeur·rice·s",
+    "addVoters": "Ajouter des électeur·rice·s",
```

```suggestion
    "addVoters": "Ajouter des électeurices",
```

According to https://www.blog.eninclusif.fr/post/%C3%A9lectrice-et-%C3%A9lecteur-en-%C3%A9criture-inclusive-epicene

---

Les deux sont possibles, mais "electeurices" n'est pas encore accepté dans les dictionnaires français. Raison pour laquelle je préfère garder les traits d'union.